.github/workflows/ci.yml

```diff
@@ -16,6 +16,7 @@ env:
   CI: true
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   REPO: ${{ github.repository }}
+  PARALLEL_TEST_FIRST_IS_1: true
```

Comment:
Are we using this?

Reply:
yes, it's used internally by `parallel_tests`. We need it so that the first node uses the index `1` instead of empty `''`

Reply:
https://github.com/grosser/parallel_tests#add-to-configdatabaseyml

Reply:
will remove the test commit then